Mixpanel/Mixpanel.m

```diff
@@ -96,8 +96,6 @@ - (instancetype)initWithToken:(NSString *)apiToken launchOptions:(NSDictionary *
         self.telephonyInfo = [[CTTelephonyNetworkInfo alloc] init];
 #endif
 #endif
-        MPSetLoggingEnabled(YES);
```

Comment:
I don't have the full context but according to the title of the pr `turn logging off by default as per documentation`, don't you want to do `MPSetLoggingEnabled(NO)` instead of removing the call here and the definition? Our users might want to turn it on later.